internal/lib/container_server.go

```diff
@@ -281,6 +281,9 @@ func (c *ContainerServer) LoadSandbox(ctx context.Context, id string) (sb *sandb
 		return sb, err
 	}
 
+	// We should restore the infraContainer to the container state store
+	c.AddInfraContainer(scontainer)
```

Comment:
**To the reviewer:**

Updated for CRI-O 1.25 to resolve the following function signature difference:

```console
# github.com/cri-o/cri-o/internal/lib
Error: internal/lib/container_server.go:285:27: too many arguments in call to c.AddInfraContainer
	have (context.Context, *oci.Container)
	want (*oci.Container)
```